src/content/en/tools/workbox/_shared/workbox-sw-cdn-url.html

```diff
@@ -0,0 +1 @@
+https://storage.googleapis.com/workbox-cdn/releases/3.0.0-alpha.1/workbox-sw.js
```

Comment:
what's this for?

Reply:
Its so the docs can tell the user to go `importScripts('https://storage.....');` and start using Workbox.

The gulp task generates this and it's included in any page that needs it.

Reply:
OK - cool, thanks!